server/src/main/java/com/workshop/domain/ActvityDictionary.java

```diff
@@ -0,0 +1,18 @@
+package com.workshop.domain;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "ACTIVITY_DICTIONARY")
+public class ActvityDictionary {
+
+    @Id
+    private String activityType;
```

Comment:
1. should we use activityType as @Id? seems like not unique value, but i don't really remember what this filed should represent so just pointing this one out
2. I'd suggest to create activityType as some ENUM instead of String